web/templates/account/edit_account.html

```diff
@@ -1,184 +1,363 @@
+                                            </p>
+                                            <h3>Example Request</h3>
+                                            <p>
+                                                You can use our API in your preferred programming language. Here is an example request in curl. We currently have a <a href="https://pypi.org/project/colivara-py/" target="_blank"> Python SDK </a>, and are working on SDKs for other languages.
+                                            </p>
+                                            <pre class="overflow-x-auto text-sm text-gray-500 bg-gray-900 p-5 rounded-2xl shadow-lg shadow-black/[0.03] not-prose">
+                                            <code class="font-mono">
+                                            <span class="text-purple-500">curl</span> <span class="text-blue-500">-L</span> \
+                                            <span class="text-purple-500">-X POST</span> \
+                                            <span class="text-purple-500">-H</span> <span class="text-blue-500">'Authorization: Bearer &lt;token&gt;'</span> \
+                                            <span class="text-purple-500">-H</span> <span class="text-blue-500">'Content-Type: application/json'</span> \
+                                            <span class="text-purple-500">'https://api.colivara.com/v1/search/'</span> \
+                                            <span class="text-purple-500">-d</span> <span class="text-blue-500">'{"query": "What is 1 + 1?"}'</span>        
+                                            </code></pre>
+
+                                            <h2 data-scrollspy-target id="api-reference" >Accessing the APIs</h2>
+                                            <p>
+                                                Our API have Swagger and OpenAPI schema available at <a href="https://api.colivara.com/v1/docs" target="_blank"> https://api.colivara.com/v1/docs </a>
+                                            </p>
+
+                                            <h2 data-scrollspy-target id="settings">Settings</h2>
+                                            <form action="{% url 'edit_account' %}" method="post">
+                                                {% csrf_token %}
+                                                {% if form.errors %}
+                                                  {% include "partials/form_errors.html" %}
+                                                {% endif %}
+                                                <div class="space-y-4">
+                                                    <div>
+                                                        <label class="block text-sm text-gray-700 font-medium mb-1" for="name">First Name</label>
+                                                        <input id="first_name" name="first_name" class="form-input py-2 w-full" type="text" value="{{ request.user.first_name }}" />
```

Comment:
should we remove the first/last name fields from the edit account since they are not part of the sign up form and will always be null for new users? the sign up form only asks for an email and password currently.